### PR TITLE
Fix deployment issues

### PR DESCRIPTION
**Pull request by ben-kaufman**

### CODE REVIEW

File: ops/graph-deploy.js [hunk 1]
```diff
@@ -15,22 +15,21 @@ async function deploy (opts = {}) {
...
-  // if (result[0] === 1) {
-  //   console.error(`Create failed! ${msg}`)
-  // }
-  if (msg.toLowerCase().indexOf('error') > 0) {
-    if (msg.match(/subgraph already exists/)) {
-      // the subgraph was already created before -we're ok
-      console.log('subgraph already exists - deploying a new version')
-    } else {
-      console.error(`Create failed! ${msg}`)
+  if (graphNode !== 'https://api.thegraph.com/deploy/') {
```
Comment (orenyodfat) [inline ops/graph-deploy.js]: what is the 'else' of this condition ?

Comment (ben-kaufman) [reply ops/graph-deploy.js]: There should be no else.

Comment (orenyodfat) [reply ops/graph-deploy.js]: why this is needed ?

Comment (ben-kaufman) [reply ops/graph-deploy.js]: Because you can't create subgraph which is on The Graph's hosted services, so only if it's not that you should create it.

Comment (orenyodfat) [reply ops/graph-deploy.js]: oh, ok ..need to be able to do that.

**Pull request merged**